languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml

```diff
@@ -92061,6 +92068,11 @@ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301, USA
                     </and>
                     <token regexp="yes">lui|y|[nv]ous|leur|[mt]oi</token>
                 </antipattern>
+                <antipattern>
```

Comment:
discussion: so here (IMP_PRON[12]) the example would have been "On vous recommande de vous adresser vers le magasin ou vous l'avez acheté." (indeed it's generating really weird things Oo)